apps/cheshire-cat-ai/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.7'
+
+services:
+  cheshire-cat-core:
+    networks:
+      - cheshire-net
+    image: ghcr.io/cheshire-cat-ai/core:latest
+    container_name: cheshire_cat_core
+    ports:
+      - ${APP_PORT}:80
+    environment:
+      - PYTHONUNBUFFERED=1
+      - WATCHFILES_FORCE_POLLING=true
+    volumes:
```

Comment:
In the volumes bellow instead of using ```${APP_DATA_DIR}/data/cheshire-cat-ai/static:/app/cat/static``` please use ```${APP_DATA_DIR}/data/static:/app/cat/static```